pallets/order-book/src/tests/extrinsics.rs

```diff
@@ -595,3 +595,122 @@ fn should_place_a_lot_of_orders() {
         }
     });
 }
+
+#[test]
+fn should_not_cancel_unknown_limit_order() {
```

Comment:
maybe also add test on non-existent order book

Reply:
I thought about it, but it always return UnknownLimiOrder error, because it is checked first. And if order book is not exist, obviously the limit order in the order book is not exist too